img/foxsi_image_det.pro

```diff
@@ -77,7 +80,7 @@ FUNCTION FOXSI_IMAGE_DET, DATA, ERANGE = ERANGE, TRANGE = TRANGE, $
   
 	if keyword_set(stop) then stop
 
-	img = reverse( img, 2 )
+	IF CDTE EQ 0 THEN img = reverse( img, 2 )
```

Comment:
What does happen if this keyword is not zero? I can't find that within the code.

Reply:
If it not zero, then we do not do this operation of flipping the image on the y-axis. This operation is a fix that was introduced for Silicon detectors but is not actually needed for CdTe.
The only way the keyword is not zero is if the user set the keyword to 1 or something else.